roles/letsencrypt/README.md

```diff
@@ -14,50 +14,24 @@ ansible-galaxy collection install t_systems_mms.letsencrypt
 
 ## http-challenge
 
-It currently uses an AWS S3 bucket to safe the hashfiles. You have to set a redirect rule in your proxy or webserver to allow the acme challenge bot to read the file, during the http-01 challenge to work:
+The validation via http-challenge currently supports the usage of a local path at the webserver and also an AWS S3 bucket to safe the hashfiles.
 
-**HaProxy(Version <=> 1.5):**
-
-```bash
-  # lets encrypt redirect
-  http-request del-header X-REWRITE
-  http-request add-header X-REWRITE %[url] if { path_beg /.well-known/acme-challenge }
-  http-request replace-header X-REWRITE ^(.well-known/acme-challenge/.*)?$ /\1 if { hdr_cnt(X-REWRITE) gt 0 }
-  http-request redirect code 301 location https://letsencrypt-challenge-bucket.s3.amazonaws.com%[hdr(X-REWRITE)] if { hdr_cnt(X-REWRITE) gt 0 }
-```
-
-**HaProxy(Version >= 1.6):**
-
-```bash
-http-request redirect code 301 location https://letsencrypt-challenge-bucket.s3.amazonaws.com%[url,regsub(^/.well-known/acme-challenge,/.well-known/acme-challenge,)] if { path_beg /.well-known/acme-challenge }
-```
-
-(can be set in frontend oder backend definition)
-
-**Apache:**
-
-```bash
-RewriteRule (\.well-known/acme-challenge.*) https://letsencrypt-challenge-bucket.s3.amazonaws.com/$1
-```
-
-**Nginx:**
-
-```bash
-rewrite (\.well-known/acme-challenge.*) https://letsencrypt-challenge-bucket.s3.amazonaws.com/$1
-```
+Please see [the corresponding README](roles/letsencrypt/README-http-challenge.md) for usage hints and variables of the HTTP-01 challenge
 
 ## dns-challenge
+
 Currently the role supports the InternetX autodns API and the Azure DNS API. Feel free to contribute with other DNS APIs.
 
-## Variables for DNS & HTTP challenge
+Please see [the corresponding README](roles/letsencrypt/README-dns-challenge.md) for variables of the DNS-01 challenge
```

Comment:
that's a relative path. Try clicking it here: https://github.com/T-Systems-MMS/ansible-collection-letsencrypt/blob/provider_for_local_path/roles/letsencrypt/README.md

Reply:
yep, missed that